packages/storage-adapters/resolve-storage-mongo/package.json

```diff
@@ -24,7 +24,9 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "jest": "^22.1.4",
-    "sinon": "^3.3.0"
+    "sinon": "^3.3.0",
+    "bluebird": "^2.9.34",
```

Comment:
Please, clarify, why you add the 'bluebird' package? Is it a syntax sugar or really necessary thing?

Reply:
In this case - because of Promise.map and Promise.delay functions only in tests. But i greatly suggest to use this promises library instead of nodejs promises:
[why faster](https://softwareengineering.stackexchange.com/questions/278778/why-are-native-es6-promises-slower-and-more-memory-intensive-than-bluebird)
[why better](https://pub.clevertech.biz/native-javascript-promises-vs-bluebird-9e58611be22)